src/Illuminate/Database/Schema/MySqlSchemaState.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Illuminate\Database\Schema;
 
-use Exception;
```

Comment:
Please re-add this as the type is still used in `executeDumpProcess`

Reply:
Thank you! It was applied.